W3ChampionsStatisticService/Hubs/WebsiteBackendHub.cs

```diff
@@ -54,8 +56,10 @@ public async Task LoadFriendListAndRequests()
     {
         var currentUser = _connections.GetUser(Context.ConnectionId)?.BattleTag;
         if (currentUser == null)
+        {
             return;
-        Friendlist friendList = await _friendRepository.LoadFriendlist(currentUser);
+        }
+        Friendlist friendList = await _friendListCache.LoadFriendList(currentUser);
```

Comment:
I think this (and the other calls in this file) should be using the handler not the cache?

Reply:
It wouldn't do anything more than call the cache anyway, so there's no need to abstract it.

Reply:
But if you think it makes more sense that way, I'll do it!

Reply:
Scratch that. You're right.